lightyear/src/client/networking.rs

```diff
@@ -66,14 +66,6 @@ impl Plugin for ClientNetworkingPlugin {
                     .chain(),
             )
             // SYSTEMS
-            .add_systems(
```

Comment:
this system was weirdly duplicated + i don't think it's necessary anymore for it to be in a different set